libexec/cli/import.help

```diff
@@ -28,8 +26,6 @@ EXAMPLES:
 
     $ gunzip -c debian.tar.gz | singularity import /tmp/Debian
```

Comment:
Maybe .img is missing here?
`gunzip -c debian.tar.gz | singularity import /tmp/Debian.img`